src/main/java/com/spotify/futures/FuturesExtra.java

```diff
@@ -159,6 +159,62 @@ public void onFailure(Throwable t) {
   }
 
   /**
+   * Represents an operation that accepts a single input argument and returns no result.
+   */
+  public interface Consumer<T> {
+
+    /**
+     * Performs this operation on the given argument.
+     * @param t the input argument
+     */
+    void accept(T t);
```

Comment:
Yeah, I thought it's nice if it behaves the same way as the onSuccess and onFailure callbacks in guava.